superset/connectors/sqla/models.py

```diff
@@ -1023,23 +1052,6 @@ def adhoc_column_to_sqla(  # pylint: disable=too-many-locals
             )
         return self.make_sqla_column_compatible(sqla_column, label)
 
-    def make_sqla_column_compatible(
```

Comment:
The `SqlaTable.make_sqla_column_compatible` method was moved to `Database.make_sqla_column_compatible` given that now the `TableColumn.table` variable can be `None`.